pkg/manager/internal.go

```diff
@@ -260,6 +263,21 @@ func (cm *controllerManager) SetFields(i interface{}) error {
 	return nil
 }
 
+// AddMetricsExtraHandler adds extra handler served on path to the http server that serves metrics.
```

Comment:
Could you extend this comment to mention that if the simple `path -> handler` mapping offered here is not enough, a new httpListener should be added as `Runnable` to the manager? (My impression is that most ppl are not aware that the manager can run anything, not just controllers)

Reply:
And probably also that this endpoint is somewhat sensitive and shouldn't be exposed publicly

Reply:
Thank you for the suggestion,  done that. Please check if I formulated the comment correctly.

Reply:
Forgot to mention that I've added the comment to the interface, not the implementation.

Reply:
sounds good, thank you